src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -69,7 +70,7 @@ const defaultProps = {
 };
 
 const MoneyRequestAction = (props) => {
-    const hasMultipleParticipants = props.chatReport.participants.length > 1;
+    const hasMultipleParticipants = lodashGet(props.chatReport, 'participants', []).length > 1;
```

Comment:
I was running to same issues now as I have the Backend adding the actions to the iou/expense reports, it was crashing the app so this is accessing the property safely. Should be better than nothing